raiden/smart_contracts/NettingChannelLibrary.sol

```diff
@@ -326,15 +314,11 @@ library NettingChannelLibrary {
 
         if (hashlock != sha3(secret))
             throw;
-
-        Participant[2] storage participants = self.participants;
-        Participant storage participant = participants[0];
-        if (participant.node_address == caller_address) {
-            participant = participants[1];
-            if (participant.node_address != caller_address) {
-                throw;
-            }
-        }
+        
+        //Check if caller_address is a participant and select her partner
+        uint8 index = 1 - index_or_throw(self, caller_address);
```

Comment:
Looks nice here. But can you perhaprs rename the `participant` variable in this functon as `partner`?